include/shell/shell_getopt.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef SHELL_GETOPT_H__
+#define SHELL_GETOPT_H__
+
+#include <getopt.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* Function initializing shell getopt module
```

Comment:
missing `.`. Usually we don't start with `Funcion..` but write something more like `Initialize shell getopt module.`